CONTRIBUTING.md

```diff
@@ -0,0 +1,14 @@
+# Contributing to napari-gui
+
+We welcome your contributions! Please see the provided steps below and never hesitate to contact us.
+
+## Steps to contribute
```

Comment:
We should include some basic instructions on how to install the development version, get started with git, etc. here.

Reply:
👍 will be adding to next changes